src/modules/emote_menu/components/Icons.jsx

```diff
@@ -39,22 +41,22 @@ function BrandedImage({src, alt, brandSrc}) {
 }
 
 export default {
-  STAR: <FontAwesomeIcon icon={faStar} />,
-  PEOPLE: <FontAwesomeIcon icon={faSmileWink} />,
-  LEAF: <FontAwesomeIcon icon={faLeaf} />,
-  ICE_CREAM: <FontAwesomeIcon icon={faIceCream} />,
-  BASKET_BALL: <FontAwesomeIcon icon={faBasketballBall} />,
-  PLANE: <FontAwesomeIcon icon={faPlane} />,
-  BOX: <FontAwesomeIcon icon={faBox} />,
-  HEART: <FontAwesomeIcon icon={faHeart} />,
-  HEART_BROKEN: <FontAwesomeIcon icon={faHeartBroken} />,
-  FLAG: <FontAwesomeIcon icon={faFlag} />,
-  CLOCK: <FontAwesomeIcon icon={faClock} />,
-  UNLOCK: <FontAwesomeIcon icon={faUnlock} />,
-  BULB: <FontAwesomeIcon icon={faLightbulb} />,
-  TWITCH: <FontAwesomeIcon icon={faTwitch} />,
-  TWITCH_GAMING: <FontAwesomeIcon icon={twitchGamingLogo} />,
-  YOUTUBE: <FontAwesomeIcon icon={faYoutube} />,
+  STAR: <Icon as={FontAwesomeSvgIcon} fontAwesomeIcon={faStar} />,
```

Comment:
would it be smarter to have a custom `FontAwesomeIcon` which wraps `Icon`?

Reply:
This is what the official rsuite docummentation advises on doing

See: https://rsuitejs.com/components/icon/#custom-svg-icon